example/templates/arrays.ts

```diff
@@ -33,6 +33,10 @@ const schema = {
               }
             }
           }
+        },
+        'time': {
+          'type': 'string',
+          'pattern': '^([0-1][0-9]|2[0-3]):[0-5][0-9]$'
```

Comment:
I'd prefer to use a format `time` instead of a pattern, even if it's not in the standard, since we use `date` currently anyway (actually there's only `date-time` available). Using a pattern can become tedious.